public/components/utils/MappingsWarning.react.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default class MappingsWarning extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    if (this.props.capiUsages == undefined) {
+      console.error("Missing capiUsages props in MappingsWarnign")
+    } else if (this.props.capiUsages > 10000) {
+      return (
+        <div className="warning-bar-small">
+          This tag has over 10,000 uses, changes to the external references could cause slowness. If you want to add an external reference but are concerned, please contact: <code>digitalcms.dev@theguardian.com</code>
```

Comment:
Would be good to show actual number rather than "over 10000", also rather than "could cause slowness" it's worth mentioning that it "could cause high demand on the tag management infrastructure". It's the same thing just a bit more management-y.